src/dev/ApiDebugOverlay.css

```
@@ -0,0 +1,498 @@
+  --api-debug-text-3: #d4d4d8;
+  --api-debug-text-4: #a1a1aa;
+  --api-debug-text-5: #71717a;
+  --api-debug-text-6: #52525b;
+  --api-debug-text-7: #3f3f46;
+  --api-debug-border: #27272a;
+  --api-debug-chip-bg: #27272a;
+  --api-debug-shadow: rgba(0, 0, 0, 0.5);
+
+  position: fixed;
+  z-index: 99999;
+  font-family: 'Inter', system-ui, sans-serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)